### DISCUSSION TITLE
Report at the end of the script

### DISCUSSION THREAD

**Discussion by VRThompson** [tomasrudh/AutoShutdownSchedule#14] [Ideas]
Hi,

Been using this script for a while and it's fantastic, would it be possible to add code to generate a report at the end of each run and email out the results?

Thanks

**Comment by tomasrudh** [comment]
Hi,

 

Yes, it would be possible, but I am not really sure how would be best, you need a SMTP server to relay the messages through, different users might want different services.

 

You could probably enable logging in the Automation Account to a Log Analytics Workspace and there setup an alert.

 

Regards,

Tomas

 

From: VRThompson ***@***.***> 
Sent: Friday, 20 May 2022 11:40
To: tomasrudh/AutoShutdownSchedule ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [tomasrudh/AutoShutdownSchedule] Report at the end of the script (Discussion #14)

 

Hi,

Been using this script for a while and it's fantastic, would it be possible to add code to generate a report at the end of each run and email out the results?

Thanks
